qiskit/primitives/base/base_estimator.py

```diff
@@ -14,9 +14,99 @@
 
 .. estimator-desc:
 
-=====================
-Overview of Estimator
-=====================
+========================
+Overview of EstimatorV2
+========================
+
+:class:`~BaseEstimatorV2` estimates expectation values of quantum circuits for provided observables.
+
+An estimator is initialized with an empty parameter set. The estimator is used to
+create a :class:`~qiskit.providers.JobV1`, via the
+:meth:`~.BaseEstimatorV2.run()` method. This method is called
+with the list of pubs (Primitive Unified Blocs).
+Pub is composed of tuple of following parameters ``[(circuit, observables, parameter_values)]``.
+
+* quantum circuit (:math:`\psi(\theta)`): (parameterized) quantum circuits
+  :class:`~qiskit.circuit.QuantumCircuit`.
+
+* observables (:math:`H_j`): a list of :class:`~.ObservablesArrayLike` classes
+  (including :class:`~.Pauli`, :class:`~.SparsePauliOp`, str).
+
+* parameter values (:math:`\theta_k`): list of sets of values
+  to be bound to the parameters of the quantum circuits
+  (list of list of float or list of dict).
+
+The method returns a :class:`~qiskit.providers.JobV1` object, calling
+:meth:`qiskit.providers.JobV1.result()` yields the
```

Comment:
I don't think sphinx knows how to remove parens?

```suggestion
the method :meth:`qiskit.providers.JobV1.result` returns
```